docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:16.04
+MAINTAINER koalaman <koalaman@gmail.com>
+
+RUN apt-get update -qy
```

Comment:
Good point, I agree that `apt-get update`  should be combined with `apt-get run`